Makefile.am

```diff
@@ -46,6 +46,13 @@ SWUPD_COMMON_SOURCES = \
 	src/scripts.c \
 	src/bundle.c
 
+SUBDIRS = dbus-wrapper
```

Comment:
So I don't love subdirs and we haven't started to use them until this point but I can understand the reasoning for doing it this way. Overall I'd still prefer to avoid adding subdirs for one piece at this point but I could be convinced otherwise.